packages/state-manager/src/sagas/publicChannels/channelsReplicated/channelsReplicated.saga.ts

```diff
@@ -24,20 +25,10 @@ export function* channelsReplicatedSaga(
   const databaseStoredChannelsIds = databaseStoredChannels.map(channel => channel.id)
   console.log({ locallyStoredChannels, databaseStoredChannelsIds })
 
-  // Removing channels from store
-  if (databaseStoredChannelsIds.length > 0) {
-    for (const channelId of locallyStoredChannels) {
-      if (!databaseStoredChannelsIds.includes(channelId)) {
-        log(`Removing #${channelId} from store`)
-        yield* put(publicChannelsActions.deleteChannel({ channelId }))
-        yield* take(publicChannelsActions.completeChannelDeletion)
-      }
-    }
-  }
-
```

Comment:
The issue ended up being that we were deleting the old general channel before we recreated it with the new ID so the check for the presence of the new channel was happening too early.  Swapping the order of operations fixes it.